Makefile

```diff
@@ -20,7 +20,7 @@ format:
 
 test:
 	@echo "Running test suite..."
-	pytest
+	pytest -v -n auto --color=yes
```

Comment:
Why do I add `-n` and `--color`?

If only `-n` option (`pytest -n auto`), test showing would be:

![Clip_20210820_151347](https://user-images.githubusercontent.com/25895405/130197223-50582b23-a37a-4f58-a7bf-f9631eaf6dd1.png)

To show details, `-v` option is necessary.
And `--color` option forces to show colors.
So final test processing (`pytest -v -n auto --color=yes`) is the following:

![Clip_20210820_151958](https://user-images.githubusercontent.com/25895405/130197453-1ea4d5d3-3ec2-4777-926d-c1bb2aca4fbe.png)